op-nat/README.md

```diff
@@ -1,54 +1,69 @@
 # Network Acceptance Tester (NAT)
 NAT is a tool to run checks against a network to determine if it is ready for production. It helps ensure your network meets all necessary requirements before deployment.
+The checks it runs are standard Go tests.
 
 ## Concepts
 
-### Validators
-NAT uses "validators" as its core building blocks. These come in three types:
+### Test Discovery
+NAT discovers tests. You simply need to provide a path to a directory containing your tests. NAT will discover all tests within that directory and also those within any subdirectories.
+
+### Test Grouping
 1. **Test** - An individual check that validates a specific aspect of your network
 2. **Suite** - A logical grouping of related tests
 3. **Gate** - A high-level collection of suites and/or tests that represent a complete validation scenario
 
-All validators are located in the `validators` directory and are written in Go.
 
 ## Contributing
 
 Please note that this project is under active development and the API may evolve. We welcome all contributions and appreciate your interest in improving NAT!
 
-### Adding a new validator
-Adding a new validator is straightforward! Create a new file in the `validators` directory using one of these Go structs: `nat.Test`, `nat.Suite`, or `nat.Gate`.
-
-Here's a template to get you started:
-
-```go
-type MyValidatorParams struct {
-    // Your validator parameters here
-}
-
-var MyValidator = nat.Test{
-    ID: "my-validator",
-    DefaultParams: MyValidatorParams{
-        // Default parameter values
-    },
-    Fn: func(ctx context.Context, log log.Logger, config nat.Config, _ interface{}) (bool, error) {
-        // Your validation logic here
-        return true, nil
-    },
-}
+### Adding a new test/suite/gate
+All tests, suites, and gates are defined in a `validators.yaml` file. The filename is not important.
```

Comment:
is there a (current) requirement for the location of these files?

Reply:
There is not. Only that there's just one of them.